portfolio/src/main/webapp/script.js

```diff
@@ -61,10 +61,35 @@ function nextPic(n) {
 
   const imageContainer = document.getElementById('gallery-img-container');
 
-  imageContainer.innerHTML = '';
+  imageContainer.innerHTML = "";
   imageContainer.appendChild(imgElement);
 }
 
+function loadPage() {
+  showForm();
+  getData();
+}
+
+function showForm() {
+  fetch('/login-status').then(response => response.text()).then((loggedIn) => {
+    
+    var loginButton = document.createElement('button');
+    loginButton.className = 'button';
+    loginButton.innerHTML = "Login";
+    loginButton.formAction = "/login";
+    loginButton.method = "GET";
+
+    if (loggedIn == 0) {
```

Comment:
0 and 1 is bad at representing the state of user login. Instead respond a JSON object in such format:
```
{
  isUserLoggedIn: true,
}
```